specification/msi/ManagedIdentity.Management/models.tsp

```diff
@@ -0,0 +1,277 @@
+   */
+  @visibility(Lifecycle.Read, Lifecycle.Create)
+  location?: string;
+
+  /**
+   * Resource tags
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
+  @visibility(Lifecycle.Read, Lifecycle.Create, Lifecycle.Update)
+  tags?: Record<string>;
+
+  /**
+   * The properties associated with the identity.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-private-usage" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
+  @Azure.ResourceManager.Private.conditionalClientFlatten
+  properties?: UserAssignedIdentityProperties;
+}
+
+/**
+ * Values returned by the List operation for federated identity credentials.
+ */
+model FederatedIdentityCredentialsListResult
+  is Page<FederatedIdentityCredential>;
+@@extension(FederatedIdentityCredentialsListResult.value,
+  "x-ms-identifiers",
+  #["id"]
+);
+
+/**
+ * The properties associated with a federated identity credential.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
+@summary("Federated identity credential properties.")
+model FederatedIdentityCredentialProperties {
+  /**
+   * The URL of the issuer to be trusted.
+   */
+  issuer: url;
+
+  /**
+   * The identifier of the external identity.
+   */
+  subject: string;
+
+  /**
+   * The list of audiences that can appear in the issued token.
+   */
+  audiences: string[];
+}
+
+/**
+ * The default resourceUri parameter type.
+ */
+@doc("The default resourceUri parameter type.")
+model ScopeParameter {
+  #suppress "@azure-tools/typespec-azure-core/no-private-usage" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
+  @path(#{ allowReserved: true })
+  @doc("The fully qualified Azure Resource manager identifier of the resource.")
+  @Azure.ResourceManager.Private.resourceParameterBaseFor(
+    [Azure.ResourceManager.CommonTypes.ResourceHome.Extension]
+  )
+  scope: string;
+}
+
+/**
+ * Base parameters for a resource.
+ * @template Resource The type of the resource.
+ */
+@Azure.ResourceManager.Private.resourceBaseParametersOf(Resource)
+model ScopeParameters<Resource extends {}> {
+  ...ApiVersionParameter;
+  ...ScopeParameter;
+}
+
+/**
+ * A list of REST API operations supported by an Azure Resource Provider. It contains an URL link to get the next set of results.
+ */
+@summary("Operations List.")
+model OperationListResult is Page<Operation>;
+@@summary(OperationListResult.value, "Operations List.");
+@@summary(OperationListResult.nextLink, "Next Link");
+@@extension(OperationListResult.value, "x-ms-identifiers", #["name"]);
+/**
+ * Details of a REST API operation, returned from the Resource Provider Operations API
+ */
+@summary("Microsoft.ManagedIdentity Operation.")
+model Operation {
```

Comment:
Do we really need this? and OperationDisplay ?

Reply:
Yes, in the original swagger there are these two models: [Operation ](https://github.com/Azure/azure-rest-api-specs/blob/0b92e4fefc254f68b25662caa0163d81b8f0983a/specification/msi/resource-manager/Microsoft.ManagedIdentity/stable/2024-11-30/ManagedIdentity.json#L821)and OperationDisplay.